builtin-func.y

```diff
@@ -708,7 +708,7 @@ body_start:	TOK_LPB c_code_begin
 				fprintf(fp_func_def, "\tif ( %s->size() != %d )\n", arg_list_name, argc);
 				fprintf(fp_func_def, "\t\t{\n");
 				fprintf(fp_func_def,
-					"\t\treporter->Error(\"%s() takes exactly %d argument(s)\");\n",
+					"\t\tzeek::emit_builtin_error(zeek::util::fmt(\"%s() takes exactly %d argument(s)\"));\n",
```

Comment:
Why the change from reporter to builtin here?

Reply:
Because the emit_builtin_error() is including location information which the plain reporter doesn't. I didn't have that initially and some of the baselines over in Zeek lost the location info and I found that sad.